### ISSUE TITLE
add install / open button for PWA VERSION

### ISSUE THREAD

**Issue by LukeberryPi** [LukeberryPi/phived#58]
this has been in the talks for a while at https://github.com/LukeberryPi/phived/issues/33. i left comments on Header.tsx that explain this issue, replicated below.

the DOWNLOAD/OPEN BUTTON logic will be implemented considering:
if the user is using the WEB VERSION
```
const isOnWeb = false;
```
until this issue is solved, this boolean must be hardcoded to false,
so that INSTALL / OPEN BUTTON doesn't show up in prod without functionality

if the user has INSTALLED the PWA VERSION
```
const hasInstalledPwa = false;
```

if they are using the WEB VERSION and don't have the PWA installed, show INSTALL BUTTON
```
const showInstallButton = isOnWeb && !hasDownloadedPwa;
```

if they are using the WEB VERSION and have the PWA installed, show OPEN BUTTON
```
const showOpenButton = isOnWeb && hasDownloadedPwa;
```

if they are using the PWA version, no button will be shown (neither INSTALL nor OPEN)

TLDR: we want to make it easier for <1024px users to install and open the PWA version of phived while accessing from the web. all of the rendering logic + components for the INSTALL and OPEN buttons is already done. we need ways to identify the booleans listed above by using PWA / Service Worker logic.

**Comment by LukeberryPi** [comment]
this didn't move forward and is not essential to the app functionality.
